### PR TITLE
Release v19.12.0

### PR DESCRIPTION
**Pull request by sjsadowski**

### CODE REVIEW

**Comment by sjsadowski** [comment]
Just need to merge the changelog and version bump, then I'll cut the release

File: scripts/pyproject.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [tool.towncrier]
 package = "sanic"
-package_dir = "."
+package_dir = ".."
```
Comment (seemethere) [inline scripts/pyproject.toml]: Was this change intentional? Not exactly versed on this particular file but didn’t really see a comment about it from the PR

Comment (sjsadowski) [reply scripts/pyproject.toml]: it was, sorry. @harshanarayana 's 'make release' was looking for the sanic package in the scripts/ dir, not in the root of the project, and towncrier was failing to execute.

**Comment by andreymal** [comment]
>from markdown to restructured text

>from RST to MD

😑

**Comment by sjsadowski** [comment]
@andreymal we're going to have to do some cleanup to get this one out the door anyway it looks like. I'll add that to the list.
